dev/Repeater/APITests/SelectionModelTests.cs

```diff
@@ -72,11 +113,12 @@ public void ValidateSelectionChangedEvent()
                 selectionModel.SelectionChanged += delegate (SelectionModel sender, SelectionModelSelectionChangedEventArgs args) 
                 {
                     selectionChangedFiredCount++;
+
+                    // Verify SelectionChanged was raised after selection state was changed in the SelectionModel
                     ValidateSelection(selectionModel, new List<IndexPath>() { Path(4) }, new List<IndexPath>() { Path() });
                 };
 
                 Select(selectionModel, 4, true);
-                ValidateSelection(selectionModel, new List<IndexPath>() { Path(4) }, new List<IndexPath>() { Path() });
```

Comment:
>ValidateSelection(selectionModel, new List<IndexPath>() { Path(4) }, new List<IndexPath>() { Path() }); [](start = 16, length = 103)

Why are we removing this?

Reply:
That check is redundant and does not cover anything that wouldn't be not covered otherwise.